<launch>
<arg name="test_mode" default="False"/>
  <node pkg="motor_driver" type="motor_driver.py" name="motor_driver" respawn="true" output="screen">
  <param name="param_path" value="$(find motor_driver)/params/motor_params.yaml"/>
  </node>
</launch>
